README.md

```diff
@@ -241,14 +241,6 @@ e.g.:
 
     Native["java/lang/System.arraycopy.(Ljava/lang/Object;ILjava/lang/Object;II)V" = function(src, srcOffset, dst, dstOffset, length) {...};
 
-If you need to implement a method in JS but you can't declare it `native` in Java, use `Override`.
```

Comment:
I'd rather keep the `override` functionality, but limit its usage.
It's  really helpful for hooking method when debugging a black-boxed midlet.